README.md

```diff
@@ -1 +1,39 @@
-##Network Service
+# Network Service 
```

Comment:
Should we name it "Simple Network Service"?

Reply:
yes, as it is the package name